tests/test_api_releases.py

```diff
@@ -2,40 +2,115 @@
 import json
 from fastapi.testclient import TestClient
 
-from main import app
+# This is a pretty delicate thing, any change could cause it to break
+# Is there a better way to make this work?
+import sys
 
-client = TestClient(app)
+sys.path.append("../src/thor")
```

Comment:
If we run `PYTHONPATH=src/thor/ poetry run pytest` this `sys.path.append` is no longer necessary.
But I guess it's not a big deal either way.

Reply:
I agree the above is pretty brittle. Also, if you checked your PYTHONPATH you might have `../src/thor` added a bunch of times now...

A few things you can consider/try:

- Add a `conftest.py` to the base repo folder. Pytest has some path detection based off conftest, and may help with this without you directly appending the path.
- As suggested in slack, we can consider  using tox. Then all future test runs will be standardized, and using `PYTHONPATH=src/thor/ poetry run pytest` wouldn't be so bad (or it can be proper path set up in there too).
- If we decide to stick with this, consider adding in a conditional to check if the path is already in there. More over, I would use the path of the current file like this:
```
import pathlib
sys.path.append(
    pathlib.Path(__file__).parent.resolve().joinpath('src/thor')
)
```
This way it's a lot more resistant, even if it looks a bit messy.

Reply:
I have in my TODO to play with tox a bit more.
@brianwu-cdis , maybe you could give us a hello world walkthrough ?